ui/app/pages/swaps/fee-card/fee-card.test.js

```diff
@@ -1,31 +1,43 @@
 import React from 'react';
-import { render } from '@testing-library/react';
 
+import { renderWithProvider } from '../../../../../test/jest';
 import { MAINNET_CHAIN_ID } from '../../../../../shared/constants/network';
-import FeeCard from './fee-card';
+import FeeCard from '.';
 
-describe('FeeCard', () => {
-  const createProps = (customProps = {}) => {
-    return {
-      primaryFee: '1 ETH',
-      secondaryFee: '2500 USD',
-      hideTokenApprovalRow: false,
-      onFeeCardMaxRowClick: jest.fn(),
-      tokenApprovalTextComponent: <></>,
-      tokenApprovalSourceTokenSymbol: 'ABC',
-      onTokenApprovalClick: jest.fn(),
-      metaMaskFee: '0.875',
-      isBestQuote: true,
-      numberOfQuotes: 6,
-      onQuotesClick: jest.fn(),
-      tokenConversionRate: 0.015,
-      chainId: MAINNET_CHAIN_ID,
-      ...customProps,
-    };
+const createProps = (customProps = {}) => {
+  return {
+    primaryFee: '1 ETH',
+    secondaryFee: '2500 USD',
```

Comment:
The `secondaryFee` prop is an object. https://github.com/MetaMask/metamask-extension/blob/16f533e6d9edf0c29b5fb103ab26bfd24d70494c/ui/app/pages/swaps/fee-card/fee-card.js#L178-L181

Reply:
An example could be 
```
  "secondaryFee": {
    "fee": "$101.98",
    "maxFee": "$112.17"
  },
```